### PR TITLE
Add user management feature for CLI

### PR DESCRIPTION
**Pull request by GDLMadushanka**
Add functionality to view, add, delete users via the CLI

### CODE REVIEW

File: cmd/cmd/userDelete.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+import (
+    "fmt"
+    "github.com/spf13/cobra"
+    "github.com/wso2/product-mi-tooling/cmd/utils"
+    "strings"
+)
+
+// Delete user command related usage info
+const deleteUserCmdLiteral = "delete"
```
Comment (arunans23) [inline cmd/cmd/userDelete.go:29]: https://github.com/wso2/product-mi-tooling/blob/master/cmd/cmd/remoteRemove.go#L28
We have defined 'remove' as the command for removing a remote a server. IMO, we should maintain one keyword for consistency. 'delete' is better than 'remove'. Either way it should be consistent. WDYT?

Comment (GDLMadushanka) [reply cmd/cmd/userDelete.go]: +1 fixed

File: cmd/cmd/userAdd.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    body := make(map[string]string)
+    body["userId"] = userId
+    body["password"] = password
+    body["isAdmin"] = isAdmin
+
+    finalUrl := utils.GetRESTAPIBase() + utils.PrefixUsers
+    res, err := utils.InvokePOSTRequest(finalUrl, headers, body)
+    fmt.Println(res)
+    if err == nil {
```
Comment (arunans23) [inline cmd/cmd/userAdd.go:97]: Here if the POST request returns a 404, it is not an error. So we cannot say that we have created successfully.

Comment (GDLMadushanka) [reply cmd/cmd/userAdd.go]: +1 fixed

**Pull request merged**